src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx

```diff
@@ -330,11 +330,11 @@ export const EditVideoForm: React.FC<EditVideoFormProps> = ({
 
   return (
     <>
-      <FormWrapper>
+      <FormWrapper onChange={handleFormChange}>
```

Comment:
This doesn't work like it used to. `handleFormChange` won't be called when a select value changes since it's not really part of a form and thus browser doesn't know about its value changing. We need to either:
a) use the previous approach
b) fix this somehow
c) use `watch` - you mentioned `React-hook-form documentation warns about performance issues`, can you link that?

Reply:
Last point of `Rules`:
https://react-hook-form.com/api/useform/watch/#main

Reply:
Is that the case for when we pass callback to `watch` as well? My guess would be that the callback option was introduced exactly for that reason - to not require the component to rerender on change

Reply:
https://codesandbox.io/s/serene-waterfall-it1me?file=/src/App.tsx

just tried it out, it seems callback gets called without a rerender of the component